src/scores.jl

```diff
@@ -490,65 +490,85 @@ function flattened_score_names(chain_data::ChainScoreData)::Array{String, 1}
 end
 
 
-function save_scores(filename::String,
-                     chain_data::ChainScoreData,
-                     score_names::Array{String,1}=String[])
-    """ Save the `scores` in a CSV file named `filename`. We iterate through
-        each state in the history of the chain and write the CSV row-by-row.
+function save_scores_to_csv(filename::String,
```

Comment:
Don't be fooled - this is essentially the same function as the previous version of `save_scores`! It's just been shortened a teensy bit but does all the same thing.